### ISSUE TITLE
please bring back "5683 assertions thus far" reporting

### ISSUE THREAD

**Issue by glycerine** [smartystreets/goconvey#209]
I loved the feature of goconvey that told me how many tests (assertions) I had done. I was a great feeling of passing a true milestone when I got past 5000 assertions, and each time I wrote a new test, seeing this number increment was truly reinforcing.

By looking at the git code from 2014-01-01, I was able to see that the printing used to come from reporting/statistics.go:34-37, the reportAssertions() function, but that file is no longer present at all in the latest goconvey.

In summary: I really liked the count of assertions checked, could you bring it back please?

Thank you.

Jason
